src/desktop_develop.js

```diff
@@ -33,6 +33,7 @@ const TYPES = ['win64', 'mac', 'linux'];
 
 const DESKTOP_GIT_REPO = 'https://github.com/vector-im/riot-desktop.git';
 const ELECTRON_BUILDER_CFG_FILE = 'electron-builder.json';
+const KEEP_BUILDS_NUM = 7; // we keep a week's worth of nightly builds
```

Comment:
Maybe two weeks...? Just thinking a bit longer of a time window could be good for bisecting a regression.

Reply:
Yeah, that should be fine